packages/system-tests/src/areas/common.ts

```diff
@@ -223,4 +223,14 @@ export class CommonActions {
   private closeCurrentNotification(): Promise<any> {
     return this.spectron.command('workbench.action.closeMessages');
   }
+
+  private extractTextFromHTML(htmlString: string[]): string[] {
```

Comment:
Function to remove html tags and return text only. I do realize the regex's might be fragile so want to know what you think about this and if there's a better way of doing it.

Reply:
Yeah. Understood. I think for the time being this is OK.

Reply:
@lcampos we might be able to use getText() method to avoid having to do the text extraction. If it works, we avoid having to use the regex to extract the text

Reply:
That does work, I'll push a new commit. Thanks @Sweetman